index.html

```diff
@@ -1820,11 +1820,12 @@ <h2>Algorithms</h2>
 
       <p>
 The algorithms defined below operate on documents represented as <dfn
-data-cite="RFC8259#section-4">JSON objects</dfn>. This specification follows
-[[JSON-LD-API]] in representing a JSON object as an [[Infra]] [=map=]. An
-<dfn class="export">unsecured data document</dfn> is a [=map=] with no
-`proof` [=map/key=]. A <dfn class="export">secured data document</dfn> is a
-[=map=] with a `proof` [=map/key=].
+data-cite="RFC8259#section-4">JSON objects</dfn>. This specification follows the
+[[[JSON-LD11-API]]] specification in representing a JSON object as an
+[[[Infra]]] [=map=]. An <dfn class="export">unsecured data document</dfn> is a
+[[[Infra]]] [=map=]. In this scenario, an <dfn class="export">unsecured data document</dfn> is a [=map=] that has not yet had the current proof added to it, but it MAY contain a
+proof value added to it by a previous process. A <dfn class="export">secured
+data document</dfn> is a [=map=] that has had the current proof(s) added to it.
```

Comment:
```suggestion
data-cite="RFC8259#section-4">JSON objects</dfn>. This specification follows the
[[[JSON-LD11-API]]] specification in representing a JSON object as an
[[[Infra]]] [=map=]. An <dfn class="export">unsecured data document</dfn> is a
[[[Infra]]] [=map=] that has not yet had the current proof added to it, but it MAY contain a
proof value added to it by a previous process. A <dfn class="export">secured
data document</dfn> is a [=map=] that has had the current proof(s) added to it.
```